modules/apigee_m10n_teams/apigee_m10n_teams.install

```diff
@@ -30,3 +30,56 @@ function apigee_m10n_teams_install() {
     user_role_grant_permissions(RoleInterface::AUTHENTICATED_ID, MonetizationTeamsInterface::DEFAULT_AUTHENTICATED_PERMISSIONS);
   }
 }
+
+/**
+ * Implements hook_requirements().
+ */
+function apigee_m10n_teams_requirements($phase) {
+  $requirements = [];
+
+  if ($phase == 'install' || $phase == 'runtime') {
+    try {
+      /** @var \Drupal\apigee_edge\SDKConnectorInterface $sdk_connector */
+      $sdk_connector = \Drupal::service('apigee_edge.sdk_connector');
+      $org_controller = \Drupal::service('apigee_edge.controller.organization');
+      /* @var \Apigee\Edge\Api\Management\Entity\Organization $organization */
+      $organization = $org_controller->load($sdk_connector->getOrganization());
+      if ($organization && $org_controller->isOrganizationApigeeX()) {
+        // AppGroup APIs are not supported in Apigee X / Hybrid orgs with monetization enabled.
+        if ($organization->getAddonsConfig() || TRUE === $organization->getAddonsConfig()->getMonetizationConfig()->getEnabled()) {
+          $url = [
+            ':url' => 'https://cloud.google.com/apigee/docs/api-platform/get-started/compare-apigee-products#unsupported-apis',
+          ];
+          $message = ($phase == 'runtime') ?
+            t("The Apigee Monetization Teams module functionality is not available for monetization enabled org on Apigee X / Hybrid and should be uninstalled, because <a href=':url' target='_blank'>AppGroup APIs are not supported in Apigee X / Hybrid orgs with monetization enabled</a>.", $url) :
+            t("The Apigee Monetization Teams module functionality is not available for monetization enabled org on Apigee X / Hybrid because <a href=':url' target='_blank'>AppGroup APIs are not supported in Apigee X / Hybrid orgs with monetization enabled</a>.", $url);
+          $requirements['apigee_edge_teams_not_supported'] = [
+            'title' => t('Apigee Edge Teams'),
+            'description' => $message,
+            'severity' => REQUIREMENT_ERROR,
+          ];
+        }
+      } else {
+        // Edge company APIs are not supported in Apigee Hybrid orgs.
+        if ($organization && !OrganizationFeatures::isCompaniesFeatureAvailable($organization)) {
```

Comment:
Why are we checking companies ?

Reply:
**Apigee Edge** has support for Companies.. I don't think you can disable Companies API. 

Maybe the check is for **Monetization Teams** module ?

Reply:
No, it is checking for hybrid feature in the Edge Org, which don't support companies API right?

Reply:
If `features.hybrid.enabled` property is in ApigeeX Org and not in Edge Org then we don't need the else part.